### PR TITLE
Add changelog for 5.17.0

### PR DESCRIPTION
**Pull request by will-moore**
Also added #431

### CODE REVIEW

**Comment by jburel** [comment]
@will-moore I have now merged #431

**Comment by jburel** [comment]
https://github.com/ome/omero-web/pull/429 to be added too

**Comment by will-moore** [comment]
Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+- Show a spinner while loading right panel [#423](https://github.com/ome/omero-web/pull/423)
+- Web API access to eventContext [#422](https://github.com/ome/omero-web/pull/422)
+- Only show hierarchy button for single Image [#413](https://github.com/ome/omero-web/pull/413)
+- Improve documentation of search default settings [#429](https://github.com/ome/omero-web/pull/429)
+
+# Deprecation & bug fixes
+
+- Mark unused Django templates as deprecated [#424](https://github.com/ome/omero-web/pull/424)
+- Fix usage of configured Favicon [#431](https://github.com/ome/omero-web/pull/431)
+- Add Python 3.10 to the testing matrix [#421](https://github.com/ome/omero-web/pull/421)
```
Comment (jburel) [inline CHANGELOG.md:16]: 421 is neither Deprecation nor bug

**Comment by jburel** [comment]
Thanks merging

**Pull request merged**